x-pack/platform/packages/shared/kbn-elastic-assistant/impl/assistant/assistant_body/empty_convo.tsx

```diff
@@ -41,7 +41,7 @@ export const EmptyConvo: React.FC<Props> = ({
         >
           <EuiFlexGroup alignItems="center" justifyContent="center" direction="column">
             <EuiFlexItem grow={false}>
-              <AssistantAnimatedIcon />
+              <AssistantBeacon backgroundColor="ghost" />
```

Comment:
Observations:
- The previous icon was ~ 32px in hight, as illustrated by the following screenshot:

![empty_prompt_icon_before](https://github.com/user-attachments/assets/2d1f904b-9a7e-493a-a27f-450b14d2a47c)

- The new icon has an unexpected white background in dark mode, as illustrated by the following screenshot:

![empty_prompt_pr](https://github.com/user-attachments/assets/91a1255d-66e3-4a29-b9b0-707437926644)

- The screenshot above illustrates that the default size of `AssistantBeacon`:

```typescript
<AssistantBeacon backgroundColor="ghost" />
```

appears to have made the icon larger than the original. For this usage, consider:

```typescript
<AssistantBeacon backgroundColor="ghost" size="xl" />
```

which appears to restore the original size, as illustrated by the following screenshot:

![empty_convo_size_xl](https://github.com/user-attachments/assets/09ee0bde-c1b3-4958-81f2-54775f9b82ca)